docs/exposing-prometheus-alertmanager-grafana-ingress.md

```diff
@@ -48,30 +41,44 @@ local kp =
       },
     },
     ingress+:: {
-      'prometheus-k8s':
-        ingress.new() +
-        ingress.mixin.metadata.withName($.prometheus.prometheus.metadata.name) +
-        ingress.mixin.metadata.withNamespace($.prometheus.prometheus.metadata.namespace) +
-        ingress.mixin.metadata.withAnnotations({
-          'nginx.ingress.kubernetes.io/auth-type': 'basic',
-          'nginx.ingress.kubernetes.io/auth-secret': 'basic-auth',
-          'nginx.ingress.kubernetes.io/auth-realm': 'Authentication Required',
-        }) +
-        ingress.mixin.spec.withRules(
-          ingressRule.new() +
-          ingressRule.withHost('prometheus.example.com') +
-          ingressRule.mixin.http.withPaths(
-            httpIngressPath.new() +
-            httpIngressPath.mixin.backend.withServiceName($.prometheus.service.metadata.name) +
-            httpIngressPath.mixin.backend.withServicePort('web')
-          ),
-        ),
+      'prometheus-k8s': {
+        apiVersion: 'extensions/v1beta1',
```

Comment:
Actually this should be `networking.k8s.io/v1` as `extensions/v1beta1 ` is being deprecated so best to use latest API in the examples.